src/patternfly/components/Content/content.scss

```diff
@@ -255,10 +276,13 @@
 
   ul {
     padding-left: var(--pf-c-content--ul--PaddingLeft);
-    margin-top: var(--pf-c-content--ul--MarginTop);
     margin-left: var(--pf-c-content--ul--MarginLeft);
     list-style: var(--pf-c-content--ul--ListStyle);
 
+    &:not(:first-child) {
```

Comment:
Since we've wrapped these `margin` declarations in `:not(:first-child)`, the styles below where we set a custom margin for nested lists isn't working since a nested list likely will be the first/only child.

<img width="1212" alt="Screen Shot 2020-04-06 at 6 43 53 PM" src="https://user-images.githubusercontent.com/35148959/78615244-9913cc00-7836-11ea-90b4-dd96af74fc74.png">

Reply:
nice catch!